File: VB/MyCardView/MyCardViewInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.Card.ViewInfo
Imports DevExpress.XtraGrid.Views.Card
Imports System.Drawing
Imports DevExpress.XtraGrid.Views.Grid.ViewInfo

Namespace MyCardView
	Public Class MyCardViewInfo
		Inherits CardViewInfo

		Public Sub New(ByVal view As DevExpress.XtraGrid.Views.Card.CardView)
			MyBase.New(view)

		End Sub
	End Class
End Namespace

File: CS/MyCardView/MyCardViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.Card.ViewInfo;
using DevExpress.XtraGrid.Views.Card;
using System.Drawing;
using DevExpress.XtraGrid.Views.Grid.ViewInfo;

namespace MyCardView
{
    public class MyCardViewInfo : CardViewInfo
    {
        public MyCardViewInfo(DevExpress.XtraGrid.Views.Card.CardView view)
            : base(view)
        {
            
        }
    }
}
